src/main/java/seedu/exercise/model/property/Date.java

```diff
@@ -119,26 +126,64 @@ public static Date getOneWeekBeforeToday() {
         LocalDate eDate;
 
         try {
-            sDate = LocalDate.parse(startDate.toString(), formatter);
-            eDate = LocalDate.parse(endDate.toString(), formatter);
+            sDate = LocalDate.parse(startDate.toString(), STANDARD_DATE_TIME_FORMATTER);
+            eDate = LocalDate.parse(endDate.toString(), STANDARD_DATE_TIME_FORMATTER);
         } catch (DateTimeParseException e) {
             return new ArrayList<>();
         }
-        days = (int) DAYS.between(sDate, eDate) + 1;
+        days = (int) ChronoUnit.DAYS.between(sDate, eDate) + 1;
```

Comment:
What does ChronoUnit do?

Reply:
Its a set of standard units for dates. Like century, month, date, year,etc... Represented as an enum inside the java class.

Read the javadocs to find more

Reply:
Noted